tests/ui/strlen_on_c_strings.stderr

```diff
@@ -0,0 +1,16 @@
+error: using `libc::strlen` on a `CString` or `CStr` value
+  --> $DIR/strlen_on_c_strings.rs:11:24
+   |
+LL |     let len = unsafe { libc::strlen(cstring.as_ptr()) };
+   |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `cstring.as_bytes().len(), you might also need to get rid of `unsafe` block in some cases`
```

Comment:
Can this comment be fixed like this?
```suggestion
   |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `cstring.as_bytes().len()`, you might also need to get rid of `unsafe` block in some cases
```

Reply:
Seems like `span_lint_and_sugg` doesn't allow that. 
What about: "help: try this (you might also need to get rid of `unsafe` block in some cases): `cstring.as_bytes().len()`" ?

Reply:
Ok, it looks good!